apiary.apib

```diff
@@ -84,11 +84,12 @@ Sample response with pagination:
             "email": "john@piedpiper.eu"
         }
     ],
-    "pagination": {
-        "count": 200,
-        "per_page": 2,
-        "current_page": 1,
-        "total_pages": 100,
```

Comment:
fyi: we dropped this property for now, because we didn't agree on naming and is just a helper anyways
might add this in the future, if it makes more sense then